### PR TITLE
<feat>backend: setup winston in the backend

### PR DESCRIPTION
**Pull request by Arielpetit**
Please @stephane-segning and @ValantineSuh review this

### CODE REVIEW

File: expense-backend/src/index.js
Comment (stephane-segning) [inline expense-backend/src/index.js:1]: This is file is auto-generated. Please remove it

File: expense-backend/src/index.ts [hunk 1]
```diff
@@ -1,29 +1,64 @@
...
+            maxSize: '20m',
+            maxFiles: '14d'
+        })
+    ],
+    format: format.combine(
+        format.timestamp(),
+        format.json()
+    ),
+    level: 'info' // Set log level
+});
```
Comment (stephane-segning) [inline expense-backend/src/index.ts:25]: Please extract this into another file

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Comment by Arielpetit** [comment]
Fixed

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please cancel all changes not related to the Backend.

**Pull request closed without merging**